### DISCUSSION TITLE
Extended functions for GET/POST APIs

### DISCUSSION THREAD

**Discussion by ghost** [FlowiseAI/Flowise#390] [Ideas]
I am more than excited about Flowise and today I tried to use content with ScrapingBot's API. Unfortunately, the GET/POST functions are not yet equipped with usernames and APIKeys today. Also, passing variables from GET to POST doesn't seem to work easily (in my example "scraper" and "responseID"). Here is the API documentation of ScrapingBot:
https://www.scraping-bot.io/web-scraping-documentation/

Would be very happy and helpful if this is usable soon!

**Comment by HenryHengZJ** [comment]
We recently release custom tool that can be used to execute GET/POST call.

Here's the docs - https://docs.flowiseai.com/tools/custom-tool

Give this a try!

**Comment by Charlotte-br560** [comment]
Excited to hear about your experience with Flowise! Integrating ScrapingBot's API sounds promising. It's unfortunate that the current functionality lacks support for usernames, API keys, and seamless variable passing from GET to POST requests. Have you considered exploring alternatives like [Crawlbase](https://crawlbase.com/)? Their robust features might align better with your needs. Looking forward to seeing enhanced functionality soon!
